app/Models/Language/Language.php

```diff
@@ -347,8 +347,7 @@ public function scopeIncludeAutonymTranslation($query)
             $priority_q = \DB::raw('(select max(`priority`) FROM language_translations WHERE language_translation_id = languages.id AND language_source_id = languages.id LIMIT 1)');
             $join->on('autonym.language_source_id', '=', 'languages.id')
               ->on('autonym.language_translation_id', '=', 'languages.id')
-              ->where('autonym.priority', '=', $priority_q)
-              ->orderBy('autonym.priority', 'desc')->limit(1);
```

Comment:
It was removed because the `orderBy` and `limit` methods do not take effect inside the join method.